vendor/k8s.io/kubernetes/pkg/kubectl/cmd/completion.go

```diff
@@ -151,6 +151,12 @@ func runCompletionZsh(out io.Writer, boilerPlate string, kubectl *cobra.Command)
 	}
 
 	zsh_initialization := `
+__kubectl_debug() {
```

Comment:
We need to manually add it here, because by defult spf's completion are generating `__<binary_name>_debug` function.

Reply:
Makes sense, just saw https://github.com/kubernetes/kubernetes/issues/61017#issuecomment-372656711 which also explains this

Reply:
Yeah, which puts us in the problematic position. I wonder if we should start writing our own completions here, instead of entirely reusing the k8s ones.